user_service/database.py

```diff
@@ -1,20 +1,22 @@
 import os
 from fastapi import FastAPI
 import motor.motor_asyncio
+from urllib.parse import quote_plus
 
 MONGO_DETAILS = "mongodb://localhost:27017" # MongoDB details for local testing database
 
 async def connect_to_mongodb(app: FastAPI):
-    uri = os.getenv("CONNECTION_STRING")
+    uri = os.getenv("CONNECTION_STRING") or connection_string_from_env()
```

Comment:
I guess you can also use
```suggestion
    uri = os.getenv("CONNECTION_STRING", connection_string_from_env())
```
but they are all the same lol